src/test/java/bio/terra/service/dataset/flight/inheritsteward/EnableInheritStewardFlightTest.java

```diff
@@ -56,9 +83,45 @@ void testParametersForSetFlagStep() {
                   (boolean) context.arguments().get(2),
                   equalTo(true));
             })) {
-      var flight = new EnableInheritStewardFlight(inputParameters, context);
-      var steps = FlightTestUtils.getStepNames(flight);
-      assertThat(steps, contains("InheritStewardSetFlagStep"));
+      new EnableInheritStewardFlight(inputParameters, context);
+    }
+  }
+
+  @Test
+  void testParametersForLockDatasetStep() {
+    try (var mockedStep =
+        mockConstruction(
+            LockDatasetStep.class,
+            (mock, context) -> {
+              assertThat((DatasetService) context.arguments().get(0), equalTo(datasetService));
+              assertThat(
+                  "The correct datasetId is passed to the step",
+                  (UUID) context.arguments().get(1),
+                  equalTo(DATASET_ID));
+              assertThat(
+                  "The correct shared lock boolean flag is passed to the step",
+                  (boolean) context.arguments().get(2),
+                  equalTo(false));
+            })) {
+      new EnableInheritStewardFlight(inputParameters, context);
```

Comment:
One problem with this approach is that it doesn't validate that the constructor was called. A fix would be to assert on the number of objects constructed by the mock:
```
      assertThat(mockStep.constructed(), hasSize(1));
```

The remaining issue is that the result of `new EnableInheritStewardFlight()` is unused which will trigger a warning. This can be suppressed:
```
      //noinspection ResultOfObjectAllocationIgnored
      new EnableInheritStewardFlight(inputParameters, context);
```
or the test could extract the step from the flight
```
      var flight = new EnableInheritStewardFlight(inputParameters, context);
      var steps = FlightTestUtils.getStepNames(flight);
      assertThat(steps, hasItem("LockDatasetStep"));
```
but that's redundant with the first test